### PR TITLE
use PostCSS 4.1 API

### LINKED ISSUE #8: Use PostCSS 4.1 API
PostCSS Plugin Guidelines are mandatory right now. Because all of rules are about ecosystem health.

### PR DESCRIPTION
**Pull request by TrySound**
fix #8

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 test/fixtures/*.actual.css
+npm-debug.log
```
Comment (MoOx) [inline .gitignore]: this should be in your global gitignore

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 1.3.0 - 2015-06-13
+
+- Added: use PostCSS 4.1 API
```
Comment (MoOx) [inline CHANGELOG.md]: It's more "Changed: upgrade to PostCSS 4.1.x"

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,4 +12,4 @@
 
 # 1.0.0 - 2014-10-04
 
-✨ Initial release from [postcss-color](https://github.com/postcss/postcss-color)
```
Comment (MoOx) [inline CHANGELOG.md:11]: Why did you touch something that is not related to the purpose of the pr...

Comment (TrySound) [inline CHANGELOG.md:11]: It's not displayed in browsers.

Comment (MoOx) [inline CHANGELOG.md:11]: Not in your browser maybe, but in good browsers they are.

screen shot 2015-06-13 at 18 38 28
screen shot 2015-06-13 at 18 38 40

**Comment by TrySound** [comment]
Also I got an error
https://gist.github.com/TrySound/8f0b6dbdfe34731f1b17
Could it be postcss-message-helpers problem?

**Comment by MoOx** [comment]
https://github.com/ianstormtaylor/css-color-function/blob/7d60b7171b45c723883aa5856b64bbe2a7bb0b1f/lib/convert.js#L44

**Comment by TrySound** [comment]
@MoOx This error interrupted watcher.

**Comment by TrySound** [comment]
@MoOx Is it gulp-postcss or postcss problem?

File: CHANGELOG.md (hunk 3 above)
Comment (TrySound) [inline CHANGELOG.md:11]: Windows, Chrome

**Comment by MoOx** [comment]
Why did you merge this ? Since when you maintain this package ?
You should ask. There is not point to merge this if this is not published. You can use your fork until this is merged and published.

**Comment by TrySound** [comment]
Sorry. I will not do it again.

**Comment by MoOx** [comment]
Also, you tagged a wrong version that doesn't have package.json up to date with the tag. :angry:
